examples/capture_circular_stream.py

```diff
@@ -40,7 +40,7 @@ def server():
             stream = conn.makefile("wb")
             filestream = FileOutput(stream)
             filestream.start()
-            picam2.encoder.output = [circ, filestream]
+            encoder.output = [circ, filestream]
```

Comment:
I suppose if we rename `picam2.encoder` to `picam2.encoders` then we could redefine `picam2.encoder` to give you the encoder object if there's exactly one, and otherwise complain. But maybe that's bending over backwards too much for backwards compatibility, it certainly feels like it's starting to get a bit fussy!